oauth-token-client/src/main/kotlin/com/gxf/utilities/spring/oauth/providers/FileTokenProvider.kt

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: Copyright Contributors to the GXF project
+//
+// SPDX-License-Identifier: Apache-2.0
+package com.gxf.utilities.spring.oauth.providers
+
+import com.gxf.utilities.spring.oauth.config.OAuthClientProperties
+import com.gxf.utilities.spring.oauth.config.condition.OAuthTokenResourceEnabledCondition
+import com.gxf.utilities.spring.oauth.exceptions.OAuthTokenException
+import java.nio.charset.Charset
+import java.util.Optional
+import org.springframework.context.annotation.Conditional
+import org.springframework.context.annotation.Primary
+import org.springframework.core.io.Resource
+import org.springframework.stereotype.Component
+
+@Component
+@Primary
```

Comment:
Volgens mij is deze `@Primary` niet nodig, omdat de Conditions elkaar uitsluiten